.gitignore

```diff
@@ -43,3 +43,6 @@ cypress/screenshots
 # Ignore the files we generate during the build and deployment process.
 origin-bucket-metadata.json
 redirects.txt
+
+# Ignore the temporary directory created for Sentry releases.
+sentry-release
```

Comment:
The name is sufficiently clear, but would `bin/sentry-release` make more sense? (Especially since there might be other build outputs we'd want to add to `.gitignore` for things like this?